README.md

```diff
@@ -164,5 +217,7 @@ Write a route in sinatra that will print "hello world" in the web browser at the
 
 Your answer:
 ```
-Replace this with your answer
+get '/oh_hello'
+  puts "hello World"
+end
```

Comment:
puts returns nil. you want either just `"hello world"` here or `return "hello world"` because of ruby methods having implicit returns